### PR TITLE
Temp updates to datasets

### PR DESCRIPTION
**Pull request by beets**

### CODE REVIEW

File: datasets/united_states.md [hunk 1]
```diff
@@ -44,7 +44,8 @@ parent: Data Sources
 
 #### [Center for Disease Control and Prevention](https://www.cdc.gov/)
 * [CDC 500 Cities: Local Data for Better Health](https://chronicdata.cdc.gov/500-Cities-Places/500-Cities-Local-Data-for-Better-Health-2019-relea/6vp6-wxuq): The 500 Cities Project datasets contain model-based small area estimates for 27 measures of chronic disease related to unhealthy behaviors (5), health outcomes (13), and use of preventive services (9) for the 500 largest cities in the US.
-* [CDC Wonder Compressed Mortality](https://wonder.cdc.gov/cmf-icd10.html): mortality counts for all US states and counties broken down by underlying cause of death, age, race, sex, and year. [CDC Wonder Data Terms of Service](https://wonder.cdc.gov/datause.html).
+* [CDC Wonder: Compressed Mortality](https://wonder.cdc.gov/cmf-icd10.html): mortality counts for all US states and counties broken down by underlying cause of death, age, race, sex, and year. [CDC Wonder Data Terms of Service](https://wonder.cdc.gov/datause.html).
```
Comment (pradh) [inline datasets/united_states.md:47]: nit: two colons look a bit odd, perhaps `CDC Wonder - Compressed Mortality:` (for Natality too) ?

File: datasets/united_states.md [hunk 2]
```diff
@@ -44,7 +44,8 @@ parent: Data Sources
...
+* [CDC Wonder: Natality](https://wonder.cdc.gov/natality.html): counts of live births occurring within the United States to U.S. residents. Counts can be obtained by a variety of demographic characteristics, such as state and county of residence, mother's race, and mother's age, and health and medical items, such as tobacco use, method of delivery, and congenital anomalies. The data are derived from birth certificates.
```
Comment (pradh) [inline datasets/united_states.md:48]: not fully sure, but wonder if we should double quote text that comes from https://wonder.cdc.gov/natality.html, or have some way to attribute source like https://screenshot.googleplex.com/8ueJjm5yAFKLdfw?

**Review by pradh — APPROVED** [review]
Thanks for the additions!

**Review by n-h-diaz — APPROVED** [review]

File: datasets/united_states.md (hunk 2 above)
Comment (beets) [reply datasets/united_states.md:48]: thanks for the catch, done!

File: datasets/united_states.md (hunk 1 above)
Comment (beets) [reply datasets/united_states.md:47]: done

**Pull request merged**